src/legacy/core_plugins/kbn_vislib_vis_types/public/components/options/metrics_axes/line_options.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+
+import { SeriesParam } from '../../../types';
+import { NumberInputOption, SwitchOption } from '../../common';
+import { SetChart } from './chart_options';
+
+interface LineOptionsParams {
+  chart: SeriesParam;
+  setChart: SetChart;
+}
+
+function LineOptions({ chart, setChart }: LineOptionsParams) {
+  return (
+    <>
+      <SwitchOption
+        label={i18n.translate('kbnVislibVisTypes.controls.pointSeries.series.showLineLabel', {
+          defaultMessage: 'Show line',
+        })}
+        paramName="drawLinesBetweenPoints"
+        value={chart.drawLinesBetweenPoints}
+        setValue={setChart}
+      />
+
+      <SwitchOption
+        label={i18n.translate('kbnVislibVisTypes.controls.pointSeries.series.showCirclesLabel', {
+          defaultMessage: 'Show circles',
+        })}
+        paramName="showCircles"
+        value={chart.showCircles}
+        setValue={setChart}
+      />
+
+      <NumberInputOption
+        label={i18n.translate('kbnVislibVisTypes.controls.pointSeries.series.lineWidthLabel', {
+          defaultMessage: 'Line width',
+        })}
+        paramName="lineWidth"
+        step={0.5}
+        min={0}
+        value={chart.lineWidth}
```

Comment:
This didn't show a value because `lineWidth` wasn't defined in Area/Bar/Line configs. But by default it draws with value [`2`](https://github.com/elastic/kibana/blob/master/src/legacy/ui/public/vislib/visualizations/point_series/line_chart.js#L30).
I added `lineWidth: 2` to the configs. Now when a new vis is created, it will define `lineWidth` as 2 by default from the config. If a user deletes the value from the input,  the chart will be drawed with default value from [line_chart.js](https://github.com/elastic/kibana/blob/master/src/legacy/ui/public/vislib/visualizations/point_series/line_chart.js#L30) (2).